### ISSUE TITLE
Update the test data documentation (Trac #710)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#834]
Recent additions to the test data folder require updates to /sasview/test/media/testdata_help.rst

Migrated from http://trac.sasview.org/ticket/710
```json
{
    "status": "closed",
    "changetime": "2016-10-08T13:55:32",
    "_ts": "2016-10-08 13:55:32.898044+00:00",
    "description": "Recent additions to the test data folder require updates to /sasview/test/media/testdata_help.rst",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-10-06T02:54:27",
    "component": "SasView",
    "summary": "Update the test data documentation",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by smk78** [comment]
Trac update at `2016/10/06 22:02:50`: 
* **smk78** commented:

In changeset da456fba6ab06ddec7ae847f352aff18d1d855d1:

```text
#!CommitTicketReference repository="sasview" revision="da456fba6ab06ddec7ae847f352aff18d1d855d1"
Major updating of the test data documentation. Closes #834
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"

**Comment by smk78** [comment]
Trac update at `2016/10/08 13:55:32`: **smk78** commented:

Minor revision to test data documentation following additional restructuring of /sasview/test. See #836
